double-conversion/strtod.h

```diff
@@ -45,6 +45,20 @@ float Strtof(Vector<const char> buffer, int exponent);
 // no leading or trailing zeros, also no lone zero, and not 'too many' digits.
 double StrtodTrimmed(Vector<const char> trimmed, int exponent);
 
+// For special use cases, the heart of the Strtof() function is also available
+// separately, it assumes that 'trimmed' is as produced by TrimAndCut(), i.e.
+// no leading or trailing zeros, also no lone zero, and not 'too many' digits.
```

Comment:
Not a huge fan of the wording, but it is a copy of `StrtodTrimmed`.

Probably would write something like the following now:
```suggestion
// Same as Strtof, but assumes that 'trimmed' is already trimmed, as if run through TrimAndCut.
// That is, 'trimmed' must have no leading or trailing zeros, must not be a lone zero, and must not have
// 'too many' digits.
```

Feel free to ignore the suggestion.
If you accept it, please also change 'StrtodTrimmed'.